documentation/book/proc-adding-volumes-to-jbod-storage.adoc

```diff
@@ -0,0 +1,72 @@
+// Module included in the following assemblies:
+//
+// assembly-storage.adoc
+
+[id='proc-adding-volumes-to-jbod-storage-{context}']
+= Adding volumes to JBOD storage
+
+This procedure describes how to add volumes to Kafka cluster configured to use JBOD storage.
+It cannot be applied to Kafka clusters configured to use any other storage type.
+While adding new volumes, you should not change configuration of any other volumes or remove them.
```

Comment:
Does the operator detect and reject this case?

Reply:
Yes. I added that to the text.